spec/features/course/experience_points_disbursement_spec.rb

```diff
@@ -37,18 +37,21 @@
       end
 
       scenario 'I can disburse experience points' do
+        approved_course_students
         unapproved_course_student
-        unawarded_student, awarded_student, awarded_zero_student = approved_course_students
         visit disburse_experience_points_course_users_path(course)
 
         fill_in 'experience_points_disbursement_reason', with: 'a reason'
 
+        student_to_award_points, student_to_set_zero, student_to_leave_blank = \
+          approved_course_students
+
         expect(page).not_to have_content_tag_for(unapproved_course_student)
-        expect(page).to have_content_tag_for(unawarded_student)
-        within find(content_tag_selector(awarded_student)) do
+        expect(page).to have_content_tag_for(student_to_leave_blank)
+        within find(content_tag_selector(student_to_award_points)) do
           find('input.points_awarded').set '100'
         end
-        within find(content_tag_selector(awarded_zero_student)) do
+        within find(content_tag_selector(student_to_set_zero)) do
```

Comment:
Actually we can remove this - this is a logic of `experience_points_disbursement`, so we should test it there. 

After that, we can keep this feature spec simple. I'm proposing something which I'll post in this PR soon.